archive_viewer/table_models/curve_model.py

```diff
@@ -96,16 +121,107 @@ def append(self, address: Optional[str] = None, name: Optional[str] = None, colo
         color : Optional[QColor], optional
             The curve's color on the plot.
         """
-        if self.rowCount() != 1:
-            self._axis_model.append()
+        if self.rowCount() == 0:
```

Comment:
Is there a reason to keep this here instead of moving the `self._axis_model.append()` into the init?

Reply:
Yes, if the user has deleted all curves. Also I believe this is completely reworked in my axes changes so I wouldnt worry too much about this specifically. Please check the other PR if you have time

Reply:
Sounds good, I'll check PR #25